### PR TITLE
Display “What’s new” page content

### PR DESCRIPTION
**Pull request by georgeschena**
### What changed

Have added a new "What's New" page to show latest changes to the application located at `/whats-new/content`

### Why did it change

So we can display the latest changes for the application

### Issue tracking

- [P4-3456](https://dsdmoj.atlassian.net/browse/P4-3456)

## Screenshots
Screenshot 2022-03-02 at 17 56 26

#### Manual testing

Has been tested in the following browsers:

- [x] Chrome
- [x] Firefox
- [ ] Edge
- [ ] Internet Explorer

### Environment variables

- [x] No environment variables were added or changed

### CODE REVIEW

File: app/whats-new/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Dependencies
+const router = require('express').Router()
+
+const contentfulService = require('../../common/services/contentful')
+
+// Define routes
+router.get('/', (req, res) => res.redirect('/whats-new/content'))
+router.get('/content', async (req, res) => {
```
Comment (thomasleese) [inline app/whats-new/index.js:8]: I wonder if there's actually a benefit of doing this as apposed to having the page on `/whats-new` directly?

File: app/whats-new/views/content.njk [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    
+    <h2 class="govuk-heading-l">
+      {{ contentTitle }}
+    </h2>
+    
+    {{ contentBody | safe }}
+    {% else %}
+    
+    {% endif %}
+{% endblock %}
```
Comment (thomasleese) [inline app/whats-new/views/content.njk:25]: It looks like there's a newline missing at the end of this file, it's flagged by GitHub as typically there would be one.

File: app/whats-new/views/content.njk [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      </h1>
+    </header>
+    
+    <h2 class="govuk-heading-l">
+      {{ contentTitle }}
+    </h2>
+    
+    {{ contentBody | safe }}
+    {% else %}
+
```
Comment (thomasleese) [inline app/whats-new/views/content.njk:23]: Might need to discuss with @SophieDodds what we want to show here when Contentful isn't available.

**Review by thomasleese — COMMENTED** [review]
Code changes look good, I've just got a few comments. I love how easy it is to get this page up and running with Contentful though!

File: app/whats-new/views/content.njk (hunk 3 above)
Comment (georgeschena) [reply app/whats-new/views/content.njk:23]: > Code changes look good, I've just got a few comments. I love how easy it is to get this page up and running with Contentful though!

Yesss greed! once Contentful is setup, its super clean!

File: app/whats-new/views/content.njk (hunk 2 above)
Comment (georgeschena) [reply app/whats-new/views/content.njk:25]: Good spot

**Comment by georgeschena** [comment]
> Code changes look good, I've just got a few comments. I love how easy it is to get this page up and running with Contentful though!

Yeahh I agree super clean once its integrated!

File: app/whats-new/index.js (hunk 1 above)
Comment (georgeschena) [reply app/whats-new/index.js]: Yeah, makes more sense to just link to `/whats-new` than to a sub file. If we have to we can extend the routing for this path but I don't see it being needed anytime soon.

File: config/nunjucks/globals.js [hunk 4]
```diff
@@ -23,6 +23,10 @@ const footerItems = [
     href: ENABLE_COMPONENTS_LIBRARY ? componentsUrl : undefined,
     text: i18n.t('components::title'),
   },
+  {
+    href: '/whats-new',
+    text: "What's new",
+  },
```
Comment (thomasleese) [inline config/nunjucks/globals.js:29]: We might want to discuss whether it's the right time to launch this link, when we don't yet have the content written for this page - I suspect we might want to launch it later.

Comment (georgeschena) [reply config/nunjucks/globals.js:29]: Makes sense, Ill remove it for now

**Review by thomasleese — APPROVED** [review]

**Pull request merged**